src/scripts.js

```diff
@@ -356,30 +356,15 @@ function populatePantry() {
   })
 }
 
+
 function populateDropdown() {
-  const ingData = ingredientsData.filter(ingredient => {
-    return typeof ingredient.name === 'string'
```

Comment:
We'd coded for that sneaky 'undefined' edge case but it was taken out of the api this morning. Because of this, we revisited the idea of using localeCompare and it works!